05-JWT-Basics/preferred/controllers/index.js

```diff
@@ -0,0 +1,22 @@
+const jwt = require('jsonwebtoken');
+
+const signToken = (name) =>
+  jwt.sign({ username: name }, process.env.JWT_SECRET, {
+    expiresIn: process.env.JWT_EXPIRES_IN,
+  });
+
+const logon = async (req, res) => {
+  const { name } = req.body;
```

Comment:
Would be good to also handle an error case here. That is, what happens if a user hits the `logon` endpoint _without_ passing in a name? We should let them know what fields are required to use this endpoint